src/OrbitBase/FileTest.cpp

```diff
@@ -424,4 +424,22 @@ TEST(File, GetFileDateModified) {
   EXPECT_LE(file_time_or_error.value() - now, absl::Seconds(1));
 }
 
-}  // namespace orbit_base
\ No newline at end of file
+TEST(File, UnicodeFileExists) {
+  constexpr const char* kUnicodeFileName = "UnicodeFileName🚀.txt";
+  std::filesystem::path file_path = orbit_test::GetTestdataDir() / kUnicodeFileName;
```

Comment:
Referring to my previous comment: If you were wrapping `kUnicodeFileName` into the `u8path` call, then it should work also without your change.

Reply:
Yeah I need to revisit this. The problem is that we'll need to change our usage of std::filesystem so that we always use u8path, I was hoping to do the conversion once at the lowest level so that all the existing callers don't need to change, but it's not the way to go. I'm wondering if we should extend std::filesystem::path so that they always treat an incoming std::string as a utf-8 encoded string. I'll close this PR for now. Thanks for your input @beckerhe .